src/main/java/app/bpartners/geojobs/endpoint/event/model/TilingTaskCreated.java

```diff
@@ -22,6 +23,12 @@ public class TilingTaskCreated extends PojaEvent {
   @JsonProperty("tilingTask")
   private TilingTask task;
 
+  private CreateFullDetection fullDetection;
```

Comment:
No. 

The FullDetection must be persisted when the request is handled, and if necessary, get it from the ZTJ attributes associated to the TilingTask, not directly here. Can you explicit why do we need it here ?